### ISSUE TITLE
🇮🇱 Hebrew Interview Series: Algorithms & LeetCode (10 episodes)

### ISSUE THREAD

**Issue by tamirdresher** [tamirdresher/techai-explained#59]
### Overview
Hebrew algorithm interview prep — LeetCode problems solved and explained in Hebrew.

### Content
- **10 video episodes** (18-20 min each, ~3.5 hours)
- **75 LeetCode problems** with Hebrew explanations
- Python + C# solutions

### Episodes
1. Arrays & Strings — Two Pointers, Sliding Window
2. Hash Maps & Sets — Frequency counting
3. Linked Lists — Reverse, Merge, Cycle detection
4. Stacks & Queues — Monotonic stack, BFS
5. Trees & BST — DFS, BFS, traversals
6. Graphs — BFS, DFS, Dijkstra, Topological Sort
7. Dynamic Programming — Memoization, Tabulation
8. Binary Search — Variations and search space
9. Backtracking — Permutations, Combinations
10. System Design Basics — Scalability, Caching

### Gumroad Product
- Individual: $14.99
- Bundle: $29.99
"@
        labels = @("content", "hebrew", "video")
    },
    @{
        title = "🇮🇱 Hebrew Interview Series: AI & ML Track (8 episodes)"
        body = @"
### Overview
Hebrew AI/ML interview preparation for data scientists and ML engineers.

### Content
- **8 video episodes** (~2.5 hours)
- **40 interview questions** with Hebrew answers
- Python code examples

### Episodes
1. יסודות ML — Supervised, Unsupervised, RL
2. Neural Networks — Perceptron, Backpropagation
3. NLP ו-Transformers — Attention, BERT, GPT
4. Computer Vision — CNN, Object Detection
5. LLMs & Prompt Engineering — Fine-tuning, RAG
6. MLOps — Deployment, Monitoring
7. שאלות קוד ML — Implement from scratch
8. System Design for ML — Recommendations, Search

### Gumroad Product
- Individual: $12.99
- Bundle: $29.99
"@
        labels = @("content", "hebrew", "video")
    },
    @{
        title = "🇮🇱 Hebrew Interview Series: System Design (6 episodes)"
        body = @"
### Overview
Hebrew system design interview preparation for senior engineers.

### Content
- **6 video episodes** (~2 hours)
- Step-by-step design walkthroughs
- Architecture diagrams

### Episodes
1. יסודות — Scalability, CAP, Load Balancing
2. תכנון URL Shortener (bit.ly)
3. תכנון מערכת צ'אט (WhatsApp)
4. תכנון News Feed (Facebook/Instagram)
5. תכנון Video Streaming (YouTube)
6. תכנון Search Engine

### Part of Bundle
- Bundle only: $29.99
"@
        labels = @("content", "hebrew", "video")
    },
    @{
        title = "🇮🇱 Gumroad: Hebrew Interview Prep Bundle — $29.99"
        body = @"
### Bundle Contents
- C# & .NET Track (10 episodes, 50 questions) — $14.99
- Algorithms & LeetCode (10 episodes, 75 problems) — $14.99
- AI & ML Track (8 episodes, 40 questions) — $12.99
- System Design (6 episodes) — included in bundle

### Total Content
- 34 video episodes (~11 hours)
- 165+ practice questions with solutions
- Mock interview scripts
- PDF cheat sheets

### Tasks
- [ ] Create Gumroad product pages
- [ ] Set up bundle pricing
- [ ] Create landing page
- [ ] Set up email sequence for buyers
